src/main/resources/org/jenkinsci/plugins/GithubSecurityRealm/config.jelly

```diff
@@ -21,7 +21,7 @@
         </f:entry>
 
         <f:entry title="OAuth Scope(s)" field="oauthScopes" help="/plugin/github-oauth/help/realm/oauth-scopes-help.html">
-            <f:textbox default="read:org" />
+            <f:textbox default="read:org,user:email" />
```

Comment:
I'm fairly clueless about jelly<->stapler.  Is there a way of calling #defaultOauthScope from the jelly template?  That would be a lot better than my kludge of hard coding the default value in two places